src/azul/plugins/metadata/hca/transform.py

```diff
@@ -320,6 +320,16 @@ class Submitter(SubmitterBase, Enum):
         'LungMAP',
         SubmitterCategory.external
     )
+    zenodo = (
+        'bd24572b-a535-5ff8-b167-0e43d7f0d4b0',
+        'Zenodo',
+        SubmitterCategory.external
+    )
+    supplementary_information = (
```

Comment:
This is an unfortunate naming choice due to the collision with "supplementary files".

Which person or institution actually submitted these?

Reply:
Submitter ID `supplementary information` replaced with `publication`.